<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="13314" started-at="2019-01-02T19:58:15Z" finished-at="2019-01-02T19:58:29Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="13314" started-at="2019-01-02T19:58:15Z" finished-at="2019-01-02T19:58:29Z">
      <class name="tests.CreateNewCandidate">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:tests.CreateNewCandidate@71c5b236]" name="beforeClass" is-config="true" duration-ms="3011" started-at="2019-01-02T19:58:15Z" finished-at="2019-01-02T19:58:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeCreateNewCandidate()[pri:0, instance:tests.CreateNewCandidate@71c5b236]" name="beforeCreateNewCandidate" is-config="true" duration-ms="631" started-at="2019-01-02T19:58:18Z" finished-at="2019-01-02T19:58:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeCreateNewCandidate -->
        <test-method status="FAIL" signature="testCreateNewCandidate()[pri:0, instance:tests.CreateNewCandidate@71c5b236]" name="testCreateNewCandidate" duration-ms="9344" started-at="2019-01-02T19:58:19Z" finished-at="2019-01-02T19:58:28Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Candidate with name Rasheed null Lehner is not found in Candidate table expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Candidate with name Rasheed null Lehner is not found in Candidate table expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at tests.CreateNewCandidate.testCreateNewCandidate(CreateNewCandidate.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateNewCandidate -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:tests.CreateNewCandidate@71c5b236]" name="afterClass" is-config="true" duration-ms="97" started-at="2019-01-02T19:58:28Z" finished-at="2019-01-02T19:58:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- tests.CreateNewCandidate -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
